x/bank/types/key.go

```diff
@@ -23,6 +23,7 @@ const (
 var (
 	// BalancesPrefix is the prefix for the account balances store. We use a byte
 	// (instead of `[]byte("balances")` to save some disk space).
+	DenomPrefix         = []byte{0x03}
```

Comment:
```suggestion
	DenomAddressPrefix         = []byte{0x03}
```

Reply:
Well the address technically isn't part of the prefix store -- the address is a key in the prefix store. Are you OK with keeping this as-is?

Reply:
It would be nice to have something a bit clearer because this could be misinterpreted as storing something like denom metadata. Address is part of the key technically even though a prefix store is used. The key itself is still `denom - address`

Reply:
The godoc for this variable is wrong.

Reply:
@aaronc ok, fair enough 👍